cfme/automate/dialogs/dialog_element.py

```diff
@@ -181,9 +181,12 @@ def create(self, element_data):
             view.fill(element)
             view.ele_save_button.click()
         view.save_button.click()
-        view.flash.assert_no_error()
-        view.flash.assert_message('{} was saved'.
```

Comment:
**REQUIRED** We can remove the specific message match here as you have it in the test below right?

Reply:
@psav had signed off, talked with @sshveta about this, she's adding the positive case flash message assertion to the existing CRUD test, and removing here. Also fixing instances where create_dialog returns a tuple and the caller doesn't unpack.

Reply:
:+1: